notebooks/byod.py

```diff
@@ -153,53 +125,76 @@ def create_cram_crai_table(table: str, listing: Iterable[str]):
-    """
+    # Upload to your bucket with gsutil
 
-with callysto.Cell("markdown"):
-    """
-    ## Organize your upload to Terra
-    Using instructions from the document above, you will use the gsutil tool to upload data from your local computer to
-    your Terra workspace. To effectively use the example here, we suggest you upload the data with the suggestions
-    below.
-    """
+    ## Install gsutil as directed by this [document](https://support.terra.bio/hc/en-us/articles/360024056512-Uploading-to-a-workspace-Google-bucket#h_01EGP8GR3G10SKRXAC7H1ENXQ3).
+    Use the option "Set up gsutil on your local computer (step-by-step install)" which will allow you to upload files 
+    from your computer directly to Terra. (Those familiar with Terra are aware that files can also be uploaded by
+    simply dragging them in to the "Files" section in the Data tab, but as `gsutil` can upload more than one file at
+    a time and has additional error handling, it is recommended to use it over dragging.) But before you can upload, 
+    we will need to perform a few quick tasks.
 
-with callysto.Cell("markdown"):
-    """
     ### Find the path to this workspace bucket
-    """
-
-with callysto.Cell("markdown"):
-    """
     Using the os package, you can print your workspace bucket path.
     """
 
 with callysto.Cell("python"):
     bucket = os.environ["WORKSPACE_BUCKET"]
     print(bucket)
-
 with callysto.Cell("markdown"):
     """
     ### Add a prefix to your bucket path to organize your data
-    In this example, we add the prefix 'test-crai-cram'. In the terminal of your computer, you will call something like:
+    In this example, we add the prefix 'my-crams'. In the terminal of your computer, you will call something like:
 
-    `gsutil cp /Users/Documents/Example.cram gs://your_bucket_info/test-crai-cram/`
-    """
+    `gsutil cp /Users/my-cool-username/Documents/Example.cram gs://your_bucket_info/my-crams/`
 
+    We will be calling what comes after your bucket info, here represented as `my-crams`, as your sudirectory. We'll be
+    using that subdirectory name later on, so let's make note of it here.
+
+    """
+with callysto.Cell("python"):
+    subdirectory = "my-crams"
 with callysto.Cell("markdown"):
     """
+    #### A note on subdirectories
+    Subdirectories are not true directories. If all files in a subdirectory are deleted, it will cease to exist. That is
+    to say, Google Cloud does not have any equivalent to empty folders. If you would like to know more, [Google has
+    documentation on its filesystem's inner workings](https://cloud.google.com/storage/docs/gsutil/addlhelp/HowSubdirectoriesWork), 
+    but most users will not need to know the details.
+
+    ## Begin the upload
+    Now that you know what you will name your subdirectory, turn back to your computer's terminal and upload the data
+    using `gsutil cp`. Remember you may have to log into to your Google account before using gsutil into your
```

Comment:
The phrasing "before using gsutil into your workspace bucket"  is difficult to follow. Please rephrase.